### DISCUSSION TITLE
Any idea to disable EP for a moment?

### DISCUSSION THREAD

**Discussion by Laytonary** [valinet/ExplorerPatcher#1831] [General]
I am using Window 11 (which has the most bug and errors when using combine with EP) so I am asking if anyone knows how to disable it until EP gets updated?. (just **disable** cause I don't want to install it again)

**Comment by peter-sari** [comment]
It does it automatically after 3 crashes.

**Comment by pyrates999** [comment]
here's how to uninstall it: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To
